practice/src/css/sections/best-place.css

```diff
@@ -1 +1,280 @@
+.place-section {
+  background-image: url(https://s3-alpha-sig.figma.com/img/6a46/9b14/71c2ad353c582a71eba56ed3e06dae89?Expires=1710115200&Key-Pair-Id=APKAQ4GOSFWCVNEHN3O4&Signature=f1e04y7hunRZwMK3kfR6sbxf9I5gFjfbaaFA4A1ivKZoaHUUqdroajaBOdrVJz0Hk9J77gTwKBaz~NjoW4nAUg7e0QfE6cWD-wHnoxx9gA8qS4fiiZK3Jnxl2YHiXqGkyI2mWF4ob0IXKO0WCGbe435gNKWJrLgq0Ehc8daojGHU5vnUFn3mRiI-qnEvh3rIuR3aBcqDKXHi7SOI7LPkaC79tz7oRpBZ-kx5KhSipWT2fTuglaheljGHaatDelrIUwgwFhktHi0NiZSFSd0Vx94bQ1ScS-CRYVCK1pfwRBxQ1kNANwmVOEvIHCP8oVy26FQg8Tf~b0Q8i9hBB-LdTA__);
+  background-color: var(--secondary-background-color);
+  background-repeat: no-repeat;
+  background-size: cover;
+  height: 350px;

```

Comment:
@JennyDoJD Pls don't set the height for your section, it should depend on your content.